sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -494,6 +496,10 @@ class tf32;
 |precision::tf32 | float | The TF32 type has a 19 bit format with one
 sign bit, 8 exponent bits (offering the same range as float), and 10
 mantissa bits (offering the same precision as sycl::half).
+|precision::sint4 | int32_t | Eight 4-bit signed integers are packed
+into 32 bits.
+|precision::uint4 | int32_t | Eight 4-bit unsigned integers are packed
```

Comment:
```suggestion
|precision::uint4 | uint32_t | Eight 4-bit unsigned integers are packed
```

This should definitely be unsigned since all the elements are unsigned.